src/template.ts

```diff
@@ -179,6 +191,11 @@ template.innerHTML = `
       transform-origin: center center;
       transform: scale(0.9);
       transition: transform 0.3s, opacity 0.3s;
+      animation-name: wiggle;
+      animation-duration: 3s;
+      animation-iteration-count: infinite;
+      animation-timing-function: linear;
```

Comment:
Would it be possible to use a different function here to make the direction change less sudden?

Reply:
Yep, will switch to `ease-in-out`.